cmd/cluster/cluster.go

```diff
@@ -42,4 +43,6 @@ func init() {
 	ClusterCmd.AddCommand(readreplica.ReadReplicaCmd)
 	readreplica.ReadReplicaCmd.PersistentFlags().StringVarP(&readreplica.ClusterName, "cluster-name", "c", "", "[REQUIRED] The name of the cluster.")
 	readreplica.ReadReplicaCmd.MarkPersistentFlagRequired("cluster-name")
+
+	ClusterCmd.AddCommand(node.NodeCmd)
```

Comment:
Can you please add this under a feature flag?

Reply:
https://yugabyte.slack.com/archives/C04G8JB4GSD/p1678852877456819?thread_ts=1678842619.217429&cid=C04G8JB4GSD

Reply:
This command just list the nodes. that's all, I don't see why we should put behind flag.

The others command(start/stop a node) when implemented should be under feature flag.